support/metrics/sets.go

```diff
@@ -218,7 +218,7 @@ func KASRelabelConfigs(set MetricsSet) []prometheusoperatorv1.RelabelConfig {
 			},
 			{
 				Action:       "drop",
-				Regex:        "apiserver_request_duration_seconds_bucket;(0.15|0.25|0.3|0.35|0.4|0.45|0.6|0.7|0.8|0.9|1.25|1.5|1.75|2.5|3|3.5|4.5|6|7|8|9|15|25|30|50)",
```

Comment:
The escaped `\` is dropped as expected 

```
$ grep -r "apiserver_request_duration_seconds_bucket;(0.15" .
./hostedcluster-c1041c3a125fee72559b/namespaces/openshift-kube-apiserver/monitoring.coreos.com/servicemonitors/openshift-kube-apiserver.yaml:      regex: apiserver_request_duration_seconds_bucket;(0.15|0.25|0.3|0.35|0.4|0.45|0.6|0.7|0.8|0.9|1.25|1.5|1.75|2.5|3|3.5|4.5|6|7|8|9|15|25|30|50)(\.0)?
``` 

from this run https://github.com/openshift/cluster-monitoring-operator/pull/2554/checks?check_run_id=36414445959